spec/smoke_tests/weird_rspec_spec.rb

```diff
@@ -86,7 +86,7 @@
   end
 
   it_behaves_like :something
-  it_should_behave_like :something
+  it_should_behave_like :something if RSpec::Core::Version::STRING < '4.0'
```

Comment:
As we not only parse those weird specs, but also actually run them with `rake spec`, we have to conditionally call `it_should_behave_like` as this alias has been removed from RSpec 4.